src/rating-summary/ratingSummary.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FC, ReactElement, useMemo } from 'react';
+import { ISummaryProp } from './types';
+import classes from './styles.module.scss';
+import starImg from '../assets/star-grey.svg';
+import { COMMA_SEPARATED_NUMBER_REGEX, Elements, RatingIds } from '../constants';
+
+const RatingSummary: FC<ISummaryProp> = (props) => {
+  const { ratings,
+    renderLabel,
+    showCount = true,
+    showAnimation = true,
+    styles={},
+    chartColors} =props;
+
+  const getRatingLabelFromRatingId = (ratingId: string): ReactElement => {
```

Comment:
Can this be a React component?

Reply:
Do we need to make this a component, if we did it will be a very small component and props like `ratingId` and `renderLabel` has to be drilled to the child as well @arunghosh-kv